runtime/vm/description.c

```diff
@@ -155,16 +157,30 @@ calculateInstanceDescription( J9VMThread *vmThread, J9Class *ramClass, J9Class *
 #endif
 			}
 		}
+
+		ramClass->gcPriorityScanField1 = ramSuperClass->gcPriorityScanField1;
+		ramClass->gcPriorityScanField2 = ramSuperClass->gcPriorityScanField2;
```

Comment:
Is there a reason to do this here?  If it's just propagating info from the super class it might be better to do it during class creation when we propagate other info from the superclass.  ie: https://github.com/eclipse/openj9/blob/a11c342e114be85c4ee09354356385f1a653de9d/runtime/vm/createramclass.cpp#L2661-L2690

Reply:
@DanHeidinga This probably makes sense. We (in GC) are less familiar with this code. @RSalman try to identify where other info is propagated.

Reply:
👍 I'll look into this